pkg/egressgateway/manager.go

```diff
@@ -190,6 +218,47 @@ func (manager *Manager) runReconciliationAfterK8sSync(ctx context.Context) {
 	}()
 }
 
+// retryEndpointIdentityLabelsResolution spawns a goroutine that retries to
+// add/update an endpoint for which we failed to resolve the identity to a set
+// of labels
+func (manager *Manager) retryEndpointIdentityLabelsResolution(ctx context.Context, wg *sync.WaitGroup) {
```

Comment:
Nit. The name of this function (and the comment) could be generalized now that handles all upsert events.